kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/RealRenderTester.kt

```diff
@@ -213,6 +213,29 @@ internal class RealRenderTester<PropsT, StateT, OutputT : Any, RenderingT>(
     return expected
   }
 
+  private inline fun <reified T : ExpectedWorker<*>> consumeExpectedWorker(
```

Comment:
This method needs kdoc explaining why it returns null, and/or maybe a different name that makes that clear.